### PR TITLE
Add multi statement support for SQLite’s Command

### PR DESCRIPTION
**Pull request by sergeymakinen**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | yes
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | see #15276 

It's required to perform upsert statements. Currently it's only supports named placeholders. Position placeholders will lead to errors. I think it can be adjusted later.

This PR has to be merged before #15276. `$commandMap` will get more custom `Command` classes in #15276.

### CODE REVIEW

File: framework/db/sqlite/Command.php [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    /**
+     * Splits the specified SQL code into individual SQL statements and returns them
+     * or `false` if there's a single statement.
+     * @param string $sql
+     * @param array $params
+     * @return string[]|false
+     */
+    private function splitStatements($sql, $params)
+    {
+        $semicolonIndex = strpos($sql, ';');
```
Comment (samdark) [inline framework/db/sqlite/Command.php]: There could be semicolon in string content:

INSERT INTO post (content) VALUES ('Hello; world!');

**Comment by samdark** [comment]
Overall looks good but special cases with `;` in strings should be tested.

File: framework/db/sqlite/Command.php (hunk 1 above)
Comment (sergeymakinen) [reply framework/db/sqlite/Command.php]: It's okay because it is a "fail fast" condition to avoid parsing at all.

**Review by samdark — APPROVED** [review]

File: framework/db/Connection.php [hunk 2]
```diff
@@ -687,12 +712,17 @@ protected function initConnection()
      */
     public function createCommand($sql = null, $params = [])
     {
+        $driver = $this->getDriverName();
+        $config = ['class' => 'yii\db\Command'];
+        if ($this->commandClass !== $config['class']) {
```
Comment (samdark) [inline framework/db/Connection.php]: Looks suspicious. Have you meant to do this check after `if (isset($this->commandMap[$driver])) {`?

**Comment by samdark** [comment]
Merged. Thank you very much!

**Pull request merged**